Makefile.rules

```diff
@@ -1268,6 +1271,27 @@ testlibgap: ${LIBGAPTESTS}
 		$(v) -A -l $(top_srcdir) -q -T --nointeract >$(v).out && \
 		diff $(top_srcdir)/$(v).expect $(v).out && ) :
 
+# test libgap's pkg-config's libgap version reporting
+tst/testpkgconfigversion: install-libgap
+ifeq ($(shell PKG_CONFIG_PATH=$(DESTDIR)$(libdir)/pkgconfig pkg-config --modversion libgap), $(PACKAGE_VERSION))
+	@echo "pkg-config reports correct version of libgap: "$(PACKAGE_VERSION)
+else
+	@echo "pkg-config does not report version of libgap, or is not installed"
+endif
+
+# test libgap's pkg-config's libgap flags fetching
+tst/testpkgconfigbuild: install
```

Comment:
This target also seems to be phony, and should drop the `tst/` prefix
```suggestion
.PHONY: testpkgconfigbuild
testpkgconfigbuild:
```

I am not sure it is good to trigger a full (re-)install here, esp. since in the one place we use it, an install already took place.

(To me, this once more suggests that this code would be easier to maintain in `dev/ci.sh`, not here. But I am happy to leave it here if you prefer -- we can always rewrite it later should we need/want to)

Reply:
I don't quite understand. I add `install` as otherwise with `install-libgap` alone you don't get headers, and adding also `install-headers`  didn't work for some reason? 

What's wrong with `install`? It doesn't get re-triggered if it was done, does it?

Reply:
The `install` target is a phony target, meaning that `make` has no concept of it "being done" or not. As such, of course it is re-triggered; if you do something like
```
make install
make testpkgconfigbuild
```
then you end up installing twice